### PR TITLE
add lorentz transform method for einstein tensor

### LINKED ISSUE #354: Add lorentz_tranform to all the classes in symbolic module
🐞 **Problem**
Many classes like `MetricTensor`, `RiemannCurvatureTensor` does not have the `lorentz_transform()` function as of now!

🎯 **Goal**
Implement it in a similar way as that in `GenericVector` class in `symbolic/vector.py`

  - [x] `GenericVector`
  - [ ] `MetricTensor`
  - [ ] `ChristoffelSymbols`
  - [ ] `RiemannCurvatureTensor`
  - [ ] `RicciTensor`
  - [ ] `StressEnergyMomentumTensor`
  - [x] `EinsteinTensor`
  - [ ] `WeylTensor`
  - [ ] `SchoutenTensor`

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by michiboo**
Fixes #354

Partial fix, just want to make sure the first pr is correct

### CODE REVIEW

**Comment by shreyasbapat** [comment]
Good thing @michiboo .

Can you join our chatroom, we can discuyss why this fails there :D

File: src/einsteinpy/symbolic/einstein.py [hunk 1]
```diff
@@ -40,6 +41,10 @@ def __init__(self, arr, syms, config="ll", parent_metric=None):
             arr=arr, syms=syms, config=config, parent_metric=parent_metric
         )
         self._order = 2
+        self._parent_metric = parent_metric
+        self._config = config
+        if isinstance(syms, (list, tuple)):
+            self.syms = syms
```
Comment (ritzvik) [inline src/einsteinpy/symbolic/einstein.py]: These lines are not required to be added, these are already handled by the parent class :)

File: src/einsteinpy/symbolic/einstein.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import sympy
-
+import numpy as np
+from sympy import simplify, tensorcontraction, tensorproduct
```
Comment (ritzvik) [inline src/einsteinpy/symbolic/einstein.py]: We may not need these after all

File: src/einsteinpy/symbolic/einstein.py [hunk 3]
```diff
@@ -90,3 +95,36 @@ def change_config(self, newconfig="ul", metric=None):
             new_tensor, self.syms, config=newconfig, parent_metric=metric
         )
         return new_obj
+
+    def lorentz_transform(self, transformation_matrix):
```
Comment (ritzvik) [inline src/einsteinpy/symbolic/einstein.py]: all the code is already there in its parent class. I was expecting something more like this.
https://github.com/einsteinpy/einsteinpy/blob/master/src/einsteinpy/symbolic/vector.py#L95-L98

**Comment by ritzvik** [comment]
@michiboo Are you still on it??

**Comment by michiboo** [comment]
> @michiboo Are you still on it??

Hi Yes, sorry for the delay, been busy lately :/

**Comment by ritzvik** [comment]
@michiboo Can you rebase the commits. Then the ‘docs’ test would pass.
After that lemme take care of the unit tests.

File: src/einsteinpy/symbolic/einstein.py [hunk 4]
```diff
@@ -90,3 +90,23 @@ def change_config(self, newconfig="ul", metric=None):
...
+
+        Returns
+        -------
+            ~einsteinpy.symbolic.tensor.BaseRelativityTensor
+                lorentz transformed tensor(or vector)
+
+        """
+        t = super(EinsteinTensor, self).lorentz_transform(transformation_matrix)
+        return EinsteinTensor(
+            t, syms=self.syms, config=self._config, parent_metric=None,
```
Comment (ritzvik) [inline src/einsteinpy/symbolic/einstein.py]: `return EinsteinTensor(t.tensor(), syms=.......)`

`t` is a `BaseRelativityTensor` object while `t.tensor()` returns a sympy object.

**Comment by ritzvik** [comment]
@michiboo Merging your first PR!
good work 🎉

**Pull request merged**